xapian-core/tests/api_closedb.cc

```diff
@@ -488,7 +488,7 @@ DEFINE_TESTCASE(closedb10, writable && metadata) {
 }
 
 DEFINE_TESTCASE(closedb11, remote) {
-	Xapian::WritableDatabase db(get_writable_database());
+	Xapian::WritableDatabase db = get_writable_database();
```

Comment:
You didn't need to change this, so it's causing the commit to be about more than it needs to be. You can create a fixup that undoes just this change.